### PR TITLE
#1 Should add JavaDoc for public and protected methods and fields

### PR DESCRIPTION
**Pull request by duergner**
Nothing more than already written on the subject - simply adding JavaDoc to public and protected members

### CODE REVIEW

File: src/main/java/org/zalando/automata/aws/AwsEnvironmentPostProcessor.java [hunk 1]
```diff
@@ -16,53 +16,57 @@
...
 	// Before ConfigFileApplicationListener so values there can use these ones
 	private int order = ConfigFileApplicationListener.DEFAULT_ORDER - 1;
 
 	/**
+	 * Set the order for determining when the {@link EnvironmentPostProcessor} should be executed. If
+	 * not set the order will default to {@link ConfigFileApplicationListener} DEFAULT_ORDER - 1.
 	 *
-	 * @param order
+	 * @param order  The int value for the order to set. See {@link Ordered} for more details.
    */
```
Comment (jbellmann) [inline src/main/java/org/zalando/automata/aws/AwsEnvironmentPostProcessor.java]: Missing spaces here, and some more below. Can you fix this please?

**Review by jbellmann — CHANGES_REQUESTED** [review]
Well done. I added a comment regarding spaces for javadocs.

File: src/main/java/org/zalando/automata/aws/AwsEnvironmentPostProcessor.java (hunk 1 above)
Comment (duergner) [reply src/main/java/org/zalando/automata/aws/AwsEnvironmentPostProcessor.java]: What would you change here? This one should have two spaces as recommended by http://blog.joda.org/2012/11/javadoc-coding-standards.html

I added the missing double spaces on the return statement below (and changed both param and return to start lower case

**Comment by duergner** [comment]
@jbellmann are you fine with the changes I did or is there anything else that should be changed? Would like to get it ready for release.

**Comment by jbellmann** [comment]
👍

**Comment by duergner** [comment]
👍

**Pull request merged**